boards/FSM/Inc/Io/Io_SharedFreqOnlyPwmInputConfig.h

```diff
@@ -1,3 +1,3 @@
 #pragma once
 
-#define MAX_NUM_OF_FREQ_ONLY_PWM_INPUTS 2
```

Comment:
Unrelated to the PR but increasing this due to the addition of Io_WheelSpeedSensor_Init